File: 7KYU/Exes and Ohs/solution.rs
fn xo(s: &'static str) -> bool {
  println!("{}", s);
  let exes = s.chars().fold(0, |acc, x| if x == 'x' || x == 'X' {acc + 1} else {acc});
  let ohs = s.chars().fold(0, |acc, x| if x == 'o' || x == 'O' {acc + 1} else {acc});
  return exes == ohs;
}

File: 7KYU/Exes and Ohs/solution.cpp
bool XO(const std::string& str)
{
  int exes = 0;
  int ohs = 0;
  
  for(auto i = 0; i < str.length(); i++) {
    if(str[i] == 'o' || str[i] == 'O')
      ohs++;
    if(str[i] == 'x' || str[i] == 'X')
      exes++;
  }
  return exes == ohs;
}